diktat-maven-plugin/build.gradle.kts

```diff
@@ -48,3 +50,27 @@ publishing {
     }
 }
 configurePublications()
+
+publishing {
+    publications {
+        withType<MavenPublication> {
+            pom {
+                withXml {
+                    val dependencyNodes = asElement().getElementsByTagName("dependency")
+                    for (i in 0 until dependencyNodes.length) {
```

Comment:
it didn't help -- gradle generates `*.module` which has info that dependencies are runtime

Reply:
the problem is in plugin.xml inside in jar. it's generated by `maven-plugin-development`

Reply:
funny, but problem is that dependencies were specified. they should be excluded from `plugin.xml`